src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,13 @@
+.footer {
+  height: 200px;
+  background: url(/images/footer.png) center no-repeat;
+  background-size: cover;
+
+  @include onDesktop {
+    height: 555px;
+  }
```

Comment:
you can remove this, because after the tablet it will inherit. 

btw, in moke-up on tablet footer height is 480px)